build.gradle.kts

```diff
@@ -13,6 +13,10 @@ plugins {
     id("org.jetbrains.dokka") apply true
 }
 
+allprojects {
+    group = "com.github.readium.kotlin-toolkit"
```

Comment:
This allows to use the same dependency names as JitPack ones when using `includeBuild`. I also renamed the sub-projects to reflect the artifact IDs used with JitPack.